src/FixerFactory.php

```diff
@@ -209,18 +209,14 @@ public function hasRule(string $name): bool
      */
     private function getFixersConflicts(FixerInterface $fixer): array
     {
-        static $conflictMap = [
+        return [
             'blank_lines_before_namespace' => [
                 'no_blank_lines_before_namespace',
                 'single_blank_line_before_namespace',
             ],
             'no_blank_lines_before_namespace' => ['single_blank_line_before_namespace'],
             'single_import_per_statement' => ['group_import'],
-        ];
-
-        $fixerName = $fixer->getName();
-
-        return \array_key_exists($fixerName, $conflictMap) ? $conflictMap[$fixerName] : [];
+        ][$fixer->getName()] ?? [];
```

Comment:
When we drop php 7.4, we should use `match` expressions in those cases:
https://3v4l.org/W6CA0